src/components/ReportPopUp.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from "react";
+import { ReportBody } from "../types";
+import {
+  Badge,
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Divider,
+  FormControl,
+  FormLabel,
+  Heading,
+  Stack,
+  Text,
+  Textarea,
+  VStack,
+  useToast,
+} from "@chakra-ui/react";
+import { createReport } from "../services/report";
+
+interface ReportPopUpProps {
+  associated_id: string;
+  reporter_user_id: number;
+  onClose: () => void;
+}
+
+const ReportPopUp: React.FC<ReportPopUpProps> = ({
+  associated_id,
+  reporter_user_id,
+  onClose,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const toast = useToast();
+
+  const [formData, setFormData] = useState<ReportBody>({
+    associated_id: associated_id,
+    reporter_user_id: reporter_user_id,
+    type: "interview",
+    reason: "",
+  });
+
+  const handleChange = (
+    e: React.ChangeEvent<
+      HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement
+    >
+  ) => {
+    const { name, value } = e.target;
+    setFormData({
+      ...formData,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    try {
+      const res = await createReport(formData);
+
+      toast({
+        title: "Report submitted successfully",
+        status: "success",
+        duration: 3000,
+        isClosable: true,
+      });
+
+      console.log(res);
+    } catch (err) {
+      console.error(err);
```

Comment:
As elijah said should use devPrint for all these console.errors and console.logs